spinalcordtoolbox/deepseg/models.py

```diff
@@ -77,6 +78,24 @@
         "contrasts": ["t2"],
         "default": False,
     },
+    "model_seg_ms_sc_mp2rage": {
+        "url": [
+            "https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20211223/model_seg_ms_sc_mp2rage.zip"
+        ],
+        "description": "Segmentation of spinal cord on MP2RAGE data from MS participants.",
+        "contrasts": ["mp2rage"],
+        "default": False,
+    },
+    "model_seg_ms_lesion_mp2rage": {
+        "url": [
+            "https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20211223/model_seg_ms_lesion_mp2rage.zip"
+        ],
+        "description": "Segmentation of multiple sclerosis lesions on cropped MP2RAGE spinal cord data. To crop the "
+                       "data you can first segment the spinal cord using the model 'model_seg_ms_lesion_mp2rage' and "
```

Comment:
Ah! I think this should be referring to the other model, no? 

```suggestion
                       "data you can first segment the spinal cord using the model 'model_seg_ms_sc_mp2rage' and "
```

Reply:
oh boy! thank you so much for catching this!!!